src/playbacks/hls/hls.js

```diff
@@ -119,6 +133,22 @@ export default class HLS extends HTML5VideoPlayback {
     this.trigger(Events.PLAYBACK_SETTINGSUPDATE)
   }
 
+  onError(evt, data) {
+    if (data && data.fatal) {
+      switch (data.type) {
+        case Hls.ErrorTypes.NETWORK_ERROR:
+          hls.startLoad()
```

Comment:
@ronag I'm not sure I haven't looked, but that is something that hlsjs has control over so if it changed in the future then it would be our fault. If we have a limit though then I think a few repeated attempts wouldn't really matter.